pallets/schemas/src/lib.rs

```diff
@@ -42,6 +51,10 @@ pub mod pallet {
 		// Maximum number of schemas that can be registered
 		#[pallet::constant]
 		type MaxSchemaRegistrations: Get<SchemaId>;
+
+		// TODO: Definition of Currency/Currency Adaptors for MRC etc.
+		// Issue: https://github.com/LibertyDSNP/mrc/issues/77
+		type Currency: Currency<Self::AccountId>;
 	}
```

Comment:
For now using current currency default will need changes according to mrc requirements

Reply:
I dont see this being used

Reply:
It is used to define type BalanceOf right above

Reply:
I think we should remove these dependencies from here.

Reply:
Done